slidedeck/src/slides/02-react-basics/00-react-basics.pug

```diff
@@ -91,143 +91,104 @@ section
 
 section
   script(type="text/plain", class="language-jsx").
-    import ReactDOM from 'react-dom';
-
-    const Hello = props => {
-      return <h1>Hello {props.name}!</h1>;
+    const Hello = () => {
+      return <h1>Hello World!</h1>;
     };
 
-    const container = document.getElementById('root');
```

Comment:
Removed this to highlight the "simplicity" of a React component, I can re-add if we think it is valuable.

Reply:
I'd like to make sure we mention the bootstrapping process at least once *somewhere* in the slides, but I have no problem tucking it away elsewhere